<launch>
   <include file="$(find robot_vision)/launch/dul_gsteamer.launch"/>

   <include file="$(find base_com)/launch/bringup.launch"/>


   <param name="/robot1_ip" type="str" value="" />
   <param name="/robot2_ip" type="str" value="192.168.1.147" />
   <param name="/robot3_ip" type="str" value="192.168.1.134" />
   <param name="/local_ip" type="str" value="192.168.1.147"/>
   <param name="/laptop_ip" type="str" value="192.168.1.127" />

   
   <node pkg="path_calculate" name = "udp_send" type ="udp_client" />
   <node pkg="path_calculate" name = "udp_recv" type ="udp_server" />
   

   <node pkg="vmu931" name = "vmu931" type ="vmu931_node" output="screen" >
   <param name="/port" type="str" value="/dev/ttyACM0"/>
   </node>
   
   <node pkg="rosbag" type="record" name = "record_odom" args="-o /Desktop /odom -O path.bag" />
  
   <node pkg="path_calculate" name = "cmd_pub" type ="path_opt" output="screen" >
   <remap from="/robot3/cmd" to="/cmd_vel"/>
   <param name="/desired_h" type="double" value="0"/>     
   <!--> UNIT FOR heading rad, volecity m/s  <-->
   <param name="/desired_v0" type="double" value="0"/>     
   <param name="/motor_lim" type="double" value="0.45"/>
   <param name="/k_vij" type="double" value="0.12"/>
   <param name="/k_cij" type="double" value="0.15"/>
   <param name="/single_test" type="bool" value="true"/>
   </node>




</launch>
